tests/pom.xml

```diff
@@ -243,10 +236,172 @@
+                                <goals>
+                                    <goal>integration-test</goal>
+                                    <goal>verify</goal>
+                                </goals>
+                                <configuration>
+                                    <skipITs>${skip-infinispan}</skipITs>
+                                    <systemProperties>
+                                        <native.image.path>${test.app.name}</native.image.path>
+                                        <java.util.logging.manager>org.jboss.logmanager.LogManager</java.util.logging.manager>
+                                        <project.version>${project.version}</project.version>
+                                        <quarkus.profile>${quarkus.profile}</quarkus.profile>
+                                    </systemProperties>
+                                    <environmentVariables>
+                                        <REGISTRY_STORAGE>infinispan</REGISTRY_STORAGE>
+                                    </environmentVariables>
+                                    <reportsDirectory>${project.build.directory}/failsafe-reports/infinispan</reportsDirectory>
+                                    <summaryFile>${project.build.directory}/failsafe-reports/infinispan-failsafe-summary.xml</summaryFile>
+                                </configuration>
+                            </execution>
+                            <execution>
+                                <id>in memory test run</id>
+                                <goals>
+                                    <goal>integration-test</goal>
+                                    <goal>verify</goal>
+                                </goals>
+                                <configuration>
+                                    <skipITs>${skip-in-memory}</skipITs>
+                                    <systemProperties>
+                                      <native.image.path>${test.app.name}</native.image.path>
+                                        <java.util.logging.manager>org.jboss.logmanager.LogManager</java.util.logging.manager>
+                                        <project.version>${project.version}</project.version>
+                                        <quarkus.profile>${quarkus.profile}</quarkus.profile>
+                                    </systemProperties>
+                                    <environmentVariables>
+                                        <REGISTRY_STORAGE>inmemory</REGISTRY_STORAGE>
+                                    </environmentVariables>
+                                    <reportsDirectory>${project.build.directory}/failsafe-reports/inmemory</reportsDirectory>
+                                    <summaryFile>${project.build.directory}/failsafe-reports/inmemory-failsafe-summary.xml</summaryFile>
+                                </configuration>
+                            </execution>
```

Comment:
Do these executions run in sequence in the order we find them in the pom.xml?  If yes, let's move the in-memory run to be the first execution.  That will serve as the baseline.  The in-memory edition should be the one most likely to work. :)

Reply:
yes they run in the order we see it :), fixed